ckanext/dcatapchharvest/harvesters.py

```diff
@@ -186,3 +187,48 @@ def after_parsing(self, rdf_parser, harvest_job):
 def _derive_flat_title(title_dict):
     """localizes language dict if no language is specified"""
     return title_dict.get('de') or title_dict.get('fr') or title_dict.get('en') or title_dict.get('it') or ""  # noqa
+
+
+class SwissDCATI14YRDFHarvester(SwissDCATRDFHarvester):
+
+    def info(self):
+        info = super(SwissDCATI14YRDFHarvester, self).info()
+
+        info['name'] = 'dcat_ch_i14y_rdf'
+        info['title'] = 'DCAT-AP Switzerland I14Y RDF Harvester'
+        info['description'] = \
+            'Harvester for DCAT-AP Switzerland datasets from ' \
+            'an RDF graph designed for I14Y'
+
+        return info
+
+    def _get_guid(self, dataset_dict, source_url=None):
+        guid = super(SwissDCATI14YRDFHarvester, self).\
+            _get_guid(dataset_dict, source_url)
+
+        # get organization name
+        try:
+            dataset_organization = get_action('organization_show')(
```

Comment:
We should use the `get_action` function from `ckan.plugins.toolkit` here, not the one from `ckan.logic`. Take a look at how it's done elsewhere in this repo and do the same thing here.

Reply:
done